pkg/cvo/cvo_scenarios_test.go

```diff
@@ -37,19 +38,22 @@ func setupCVOTest(payloadDir string) (*Operator, map[string]runtime.Object, *fak
 	})
 	cvs := make(map[string]runtime.Object)
 	client.AddReactor("*", "clusterversions", func(action clientgotesting.Action) (handled bool, ret runtime.Object, err error) {
-		switch a := action.(type) {
-		case clientgotesting.GetAction:
+		switch strings.ToLower(action.GetVerb()) {
+		case "get":
```

Comment:
Good catch.  Checking for precedent upstream, I see them [switching on `GetActionImpl`, etc.][1]  Thoughts on that vs. just using the verb?

[1]: https://github.com/kubernetes/client-go/blob/1bccfc8c60977e2ce3235394daff128996818f7b/testing/fixture.go#L83-L97

Reply:
Replaced with the `Impl`s